calendar/calendar-tool/tool/src/webapp/vm/calendar/chef_calendar_viewWeek.vm

```diff
@@ -372,7 +370,7 @@
     	#set ($rowspan = $rowspan + 1)
     #end
     ## This endtime may have been fudged, if so it will need an extra row as it's being returned from getNewEvents
-    #if ($endTime != $endTimeActual)
+    #if ($endTime == $endTimeActual)
```

Comment:
@jonespm, please could you check this one: https://jira.sakaiproject.org/browse/SAK-41386?
It is related with this change. Thanks!